googlemaps/places.py

```diff
@@ -0,0 +1,203 @@
+# the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+#
+
+"""Performs requests to the Google Maps Places API."""
+
+from googlemaps import convert
+
+
+def places(client, query, location=None, radius=None, language=None,
+           min_price=None, max_price=None, open_now=False, types=None,
+           page_token=None):
+    """
+    Performs text search for places.
+
+    :param query: The text string on which to search, for example: "restaurant".
+    :type query: string
+
+    :param location: The latitude/longitude value for which you wish to obtain the
+                     closest, human-readable address.
+    :type location: string, dict, list, or tuple
+
+    :param radius: Distance in meters within which to bias results.
+    :type radius: int
+
+    :param language: The language in which to return results.
+    :type langauge: string
+
+    :param min_price: Restricts results to only those places with no less than
+                      this price level. Valid values are in the range from 0
+                      (most affordable) to 4 (most expensive).
+    :type min_price: int
+
+    :param max_price: Restricts results to only those places with no greater
+                      than this price level. Valid values are in the range
+                      from 0 (most affordable) to 4 (most expensive).
+    :type max_price: int
+
+    :param open_now: Return only those places that are open for business at
+                     the time the query is sent.
+    :type open_now: bool
+
+    :param types: Restricts the results to places matching at least one of the
+                  specified types.
+    :type types: string or list of strings
+
+    :param page_token: Token from a previous search that when provided will
+                       returns the next page of results for the same search.
+    :type page_token: string
+
+    :rtype: result dict with the following keys:
+            status: status code
+            results: list of places
+            html_attributions: set of attributions which must be displayed
+            next_page_token: token for retrieving the next page of results
+
+    """
+    params = {}
+
+    if query:
+        params["query"] = query
+    if location:
+        params["location"] = convert.latlng(location)
+    if radius:
+        params["radius"] = radius
+    if language:
+        params["language"] = language
+    if min_price:
+        params["minprice"] = min_price
+    if max_price:
+        params["maxprice"] = max_price
+    if open_now:
+        params["opennow"] = "true"
+    if page_token:
```

Comment:
I think it reads more clearly without them but I guess it's subjective. Does pep8 or our internal style guide have any say on it? Closest I could find is a vague mention in pep8 that says "use sparingly", I think spaces here would be the opposite of that. :-)